src/nft/components/details/detailsV2/DataPage.tsx

```diff
@@ -0,0 +1,55 @@
+import Column from 'components/Column'
+import Row from 'components/Row'
+import styled from 'styled-components/macro'
+import { BREAKPOINTS } from 'theme'
+
+import { DataPageDescription } from './DataPageDescription'
+import { DataPageHeader } from './DataPageHeader'
+import { DataPageTable } from './DataPageTable'
+import { DataPageTraits } from './DataPageTraits'
+
+const DataPageContainer = styled(Column)`
+  padding: 24px 64px;
+  height: 100vh;
+  width: 100%;
+  gap: 36px;
+  max-width: 1200px;
+  margin: 0 auto;
+
+  @media screen and (max-width: ${BREAKPOINTS.sm}px) {
+    padding: 24px 48px;
```

Comment:
The X padding for sm and xs screen sizes could be re-used for the Landing page but otherwise these are only for the Data Page.

Reply:
How different are they from the rest of the website? Is this something we can try to unify?

Reply:
Also, if those values are repeated in other components, maybe we can have another component, like `Padding` that handles breakpoints on its own.

Reply:
I just parsed the code base and didn't find another example where `padding: 24px 48px` was used. This may warrant a large conversation with Design to set standards but for this PR given it's the only use I lean towards current implementation.